src/idl_gen_general.cpp

```diff
@@ -587,6 +587,19 @@ std::string GenGetter(const Type &type) {
   }
 }
 
+// Returns the function name that is able to read a value of the given type.
+std::string GenGetterForLookupByKey(const Type &type,
+                                    const std::string data_buffer) {
+  auto getter = data_buffer + "." + FunctionStart('G') + "et";
```

Comment:
Now there are only 2 similar lines. Should I do it now?
Also this function will be modified for working with boolean keys.